dependabot.yaml

```diff
@@ -0,0 +1,4 @@
+version: 2
```

Comment:
Theres also a .github/dependabot.yml you could adapt 👀

Reply:
OH!